app/themes/colorPalette.js

```diff
@@ -10,6 +10,7 @@ export default {
     blueGreyMedium: '#7E98C3',
     blueGreyLight: '#A6B1BB',
     blueGrey: '#66788A',
+    bluePrimary00: '#F0F5FF',
```

Comment:
Design is working with a [new set of colors](https://www.figma.com/design/zu1iGBaggqX072VYYhYi26/Tidepool-Web---FOUNDATIONS?node-id=2423-235&node-type=canvas&t=uOLmJjPNsM9mHqGw-0) () that are number based that will need to be folded in. I'm thinking we should discuss a what naming convention you folks would prefer during FE huddle.